pkg/operator/starter.go

```diff
@@ -253,6 +265,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	)
 
+	// TODO: since we have new conditions should we remove the previous ones ?
```

Comment:
I think this is important once we decide to use the new conditions.

Reply:
Do we have new conditions ?

Reply:
The names have changed, I think no one will put me in jail for that :) ?

Reply:
then we should clean old ones up. We actually have to.